apps/salonchromaclock/salonchromaclock.star

```diff
@@ -0,0 +1,101 @@
+e2A6Pj1l+s7pAz7GPgKfep+Hvqa+It89viN+1n6Zfgf8nvs7+sv9j/i/4XnyFvFOBWABwQHlAb2BGoGzA2sDHwSZBKUHNQWNBbsGLw
+w+FUIMCQ1ZH3KTb8AX8hv5YzPcZyya0RXKCJ0VWhv6MMwmTB7WEY6GzwjfEH5vpvlM6cy2CIjgR2yIuB9pGZkX+X0UKSoyqi7qUbRT
+dHF09yzWrORZ+2e9jvGPqYy5O9tqtnJ2Z6xqbFJsY+ybuIC4qriBeIf4RfGXEnQTJAntieTE2MQ9ieNzAudsmjOc5JpUlnRjruXcor
+kX5unOy553PFk1WZB8OIWYEpeyP+WDIEJQLxhP5aduTR0T8oSbhU9FvqKNolGxt7hKPJLmnVaV9jjdO31D+miGT0Z1xjMJT1IreZEZ
+krkj801WRNberM/ZcdktOZSclJyjUg1plrQr1zC3KLdPZisrkw3keeZtyhuTh8r35CP5c/PbFWyFTNGjtFKuUA4WTC+oK3hbGFt4uE
+i9SFrUM99m/ur5IwuCFny9kLBQuLCz2Lh4WfHgIr9FuxYji1MXdy4xXVK6ZHhp8NJ9y2jLspb9UOJYUlXyannc8o5Sg9KlpUMrglc0
+lamUycturvRauWMVYZVkVe9ql9VbVn8qF5VfrHCsqK74sEa45uJXTl/VfPV5bdra3kq3yu3rSOuk626s91m/r0q9akHV0IbwDa0b8Y
+3lG19tSt50oXpq9Y7NtM3KzQM1YTXtW8y2rNvyoTaj9nqdf13LVv2tq7e+2Sba1r/dd3vzDoMdFTve75TsvLUreFdrvUV99W7S7oLd
+jxpiG7q/5n7duEd3T8Wej3ulewf2Re/ranRvbNyvv7+yCW1SNo0eSDpw5ZuAb9qb7Zp3tXBaKg7CQeXBJ9+mfHvjUOihzsPcw83fmX
++39QjrSHkr0jq/dawto22gPaG97+iMo50dXh1Hvrf/fu8x42N1xzWPV56gnSg98fnkgpPjp2Snnp1OPz3Umdx590z8mWtdUV29Z0PP
+nj8XdO5Mt1/3yfPe549d8Lxw9CL3Ytslt0utPa49R35w/eFIr1tv62X3y+1XPK509E3rO9Hv03/6asDVc9f41y5dn3m978bsG7duJt
+0cuCW69fh29u0XdwruTNxdeo94r/y+2v3qB/oP6n+0/rFlwG3g+GDAYM/DWQ/vDgmHnv6U/9OH4dJHzEfVI0YjjY+dHx8bDRq98mTO
+k+GnsqcTz8p+Vv9563Or59/94vtLz1j82PAL+YvPv655qfNy76uprzrHI8cfvM55PfGm/K3O233vuO+638e9H5ko/ED+UPPR+mPHp9
+BP9z7nfP78L/eE8/stRzjPAAAAIGNIUk0AAHomAACAhAAA+gAAAIDoAAB1MAAA6mAAADqYAAAXcJy6UTwAAAAJcEhZcwAALiMAAC4j
+AXilP3YAAAPBSURBVEiJpZVLTFxVGMe/7zzuYx4wMsAUC5TyCGML1T4iBI0llqgV07CyMXWlKzeSxqSmJi66cGFiqsaFpmtjYkyt7j
+SxTW19QRu6kKalUWKFUqhlhmEG7tzHOZ8LHsNMp0PQ/+Ym9zvnl//5f/c7F4kI/p9YxSpl76b+unDtyukv8+nswxbhgy6czPLU2OT0
+yMSdkRuLt6Z0Ji20H2/bdvDj4cS+5CaI2YnZi5+dv3P1z6WpOa5cU6AlyOCBoACcnF1lPnX6ePNzvQ9FZO5lPz165v7N6WiUW1wLDD
+j5UgcCfaF9jkq6Diq3+61Xu4aPIWJpFu6y98WJs/9M3g/VhJhgBECEBEBICISAQISGEIKPv3fm7sXRMnF+/s634xf+sKImKfIcXytV
+kuuqZ8EYwsx3l4sQWulzH14a/eZGqMrWgdJaHzoxWNfZ4Oe9UgAAAHJLzv8y5jtOAeE6wc9fXxeGAVorrY6cGup/Y2Dg5BATSEqvbI
+PCwYFLsTQ5nRodLyDsiNH1TIvy/MD1B0++2PNKLwDs7EsefHson10mrUtsACL5/tS574uy2HeoHRmA4G0Hdq4Xel4bePxYvwqUt+Ss
+IVYf3JTpX8f8bK6A6HiyOd5UjQQbDSNjgx+8fvTsu+1DfYEK/NyaIwCU0rk9k7r6ewFhR8zHelpUoLnJixsBDXvbX/hk+MhXpzpefp
+Zb5ioFgQJ/7oefCggA6H6+k1tcGBLKqWF/sv+j473vv6m9gAiQkFvG/OUR5boFRLKvtWlXAssC1hTZXoeCERAgMCmXJ29nxicKCMOW
+XQOdBJUGHzlHIXBlIBDByacu/QYbh33vS3vCsXAFBDMl4wzWGsMt8975HwFArK+obaypeA7gpsGlIJdWGscMw/t7Bja7cookLYOFbW
+QMGQIAASGDTRBE5OYcfym/6sIyrLoYD9uARbmLcnsBADzHn7k+HaQzws1FE9U1+3dz26rv6U6j9mbnaCGNnktCVEJISyQ6EsqJMdBG
+NISMMYlNh58WHFMj1zw/r5zFUOuOSghEtKtDUB3a+OaRXW162QkWMpnMgl6Yj3S2b5LFg2KWEd7xaKS10aqP85AdTXZsGQEA3DLMeM
+ysrZGxqlB7y39BADIRCctY1KivjTQ3bh1BhJzJsM1Dtr19m6yt2ToCERljpgEMq57YzaSEsn+zSiaU9rO5xZuTRDp+YA+TAgCQtAZE
+Uho5g/UpJCr5BFcRgXIXFhHRjMcKzpTrMiGV63LLJKUp8NEwgAh56Q0GRMoLkDMmikr/AuAYkPmUq72iAAAAAElFTkSuQmCC
+""")
+
+def main(config):
+    # font = config.get("font", "tb-small")
+    timezone = config.get("timezone") or "America/Chicago"
```

Comment:
To enable users to pass a timezone, you need to add a `get_schema()` function. More info [over here](https://tidbyt.dev/docs/reference/schema).

You can also get the user's device's timezone via the `$tz` parameter: `config.get("$tz", "America/Chicago")`.

Reply:
My goal is to add the timezone selection and a way to add your own custom image in v2 (just learning how to code in STAR). But this one is to allow a person to have a custom date/time clock with their own image.